src/app/lib/config/services/serbian.js

```diff
@@ -184,6 +184,16 @@ const baseServiceConfig = {
   },
   fonts: [],
   timezone: 'GMT',
+  scriptLinkVariants: {
+    lat: {
+      scriptLinkText: 'Lat',
+      scriptLinkOffscreenText: 'Latin',
+    },
+    cyr: {
+      scriptLinkText: 'Ћир',
+      scriptLinkOffscreenText: 'Cyrillic',
+    },
+  },
```

Comment:
Since we already have the `lat` and `cyr` keys on the service config, it makes sense to place the `scripLink` stuff inside their respective `keys` instead of repeating them here.
Something like 
```js
export const service = {
  lat: {
    ...baseServiceConfig,
    skipLink: {
      text: 'Lat',
      offscreenText: 'Latin'
    }
  },
  cyr: {
    ...baseServiceConfig,
    skipLink: {
      text: 'Ћир',
      offscreenText: 'Cyrillic'
    }
  },
}

Reply:
The challenge I had with structuring this as you've suggested is that I was not able to easily access the scriptLink data of the opposite variant - as the data passed through ServiceContext is variant specific. So, if the currentVariant is `lat`, I couldn't access ```skipLink: {
      text: 'Ћир',
      offscreenText: 'Cyrillic'
    }```

Reply:
UPDATE: Paired with @sadickisaac on this, and adopted the suggestion.